dart/lib/model/binary.dart

```diff
@@ -0,0 +1,78 @@
+part of keyclic_sdk_api.api;
+
+class Binary {
+  Binary({
+    this.content,
+    this.contentType,
+    this.name,
+  });
+
+  factory Binary.fromJson(Map<String, dynamic> json) {
+    if (json == null) {
+      return null;
+    }
+
+    return Binary(
+      content: json['content'],
+      contentType: json['contentType'],
+      name: json['name'],
+    );
+  }
+
+  String content;
+
+  String contentType;
```

Comment:
Content type used to be an enum
https://github.com/Keyclic/app-sdk/pull/27/files#diff-0fe0aca7dae4e9219bcb70dec9c327ec4d5d7faf5aa491173d457d5eee7af76bL42-L43